src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -257,26 +257,39 @@ Tests scanAndSimplify(Tests tests)
             BoundPattern pattern,
             ArrayBuilder<BoundPatternBinding> bindings)
         {
+            return MakeTestsAndBindings(input, pattern, out _, bindings);
+        }
+
+        private Tests MakeTestsAndBindings(
+            BoundDagTemp input,
+            BoundPattern pattern,
+            out BoundDagTemp output,
```

Comment:
nit: consider comment or more explicit parameter name (not sure what `output` refers to in this context)

Reply:
Patterns now have input and output values.  Just as line 264 describes the pattern's input, this line describes the pattern's output.  See the draft specification at [https://github.com/dotnet/csharplang/issues/2850](https://github.com/dotnet/csharplang/issues/2850); this is a value that corresponds to the pattern's *narrowed type*.

---
In reply to: [389183904](https://github.com/dotnet/roslyn/pull/42207#discussion_r389183904) [](ancestors = 389183904)